bugbot/rules/component.py

```diff
@@ -89,6 +89,36 @@ def get_bugs(self, date="today", bug_ids=[]):
         # Classify those bugs
         bugs = get_bug_ids_classification("component", bug_ids)
 
+        # Collect bugs classified as Fenix::General
+        fenix_general_bug_ids = [
+            bug_id
+            for bug_id, bug_data in bugs.items()
+            if bug_data.get("class") == "Fenix::General"
```

Comment:
I think the model will output `Fenix` instead of `Fenix::General`.

Reply:
Fixed in 1af5b0ab6b9c86c82c39142aa33404447e673e9e.